meterfs/test_meterfs_allocate.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs allocating tests group
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ *
+ * %LICENSE%
+ */
+
+#include "common.h"
+
+TEST_GROUP(meterfs_allocate);
+
+
+TEST_SETUP(meterfs_allocate)
+{
+}
+
+
+TEST_TEAR_DOWN(meterfs_allocate)
+{
+	TEST_ASSERT_EQUAL(0, file_eraseAll());
+}
+
+
+/* NOTE: not valid for hw target .Now there is no way to obtain/parametrize flash size in test */
```

Comment:
typo:
target. Now
(full stop misalligned)
Also as it's kind of multiline comment, use coding style for it.
Also I think first general comment, then note.

Reply:
Done.